### PR TITLE
[IOTDB-1491] UDTF query supported in cluster mode

### PR DESCRIPTION
**Pull request by liuminghui233**
UDTF query supported in cluster mode (no redirect)
JR: https://issues.apache.org/jira/browse/IOTDB-1491

### TODO:
Redirect version

### CODE REVIEW

File: cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterQueryRouter.java [hunk 1]
```diff
@@ -28,21 +28,21 @@
...
+import org.apache.iotdb.db.qp.physical.crud.*;
 import org.apache.iotdb.db.query.context.QueryContext;
 import org.apache.iotdb.db.query.dataset.groupby.GroupByWithValueFilterDataSet;
 import org.apache.iotdb.db.query.dataset.groupby.GroupByWithoutValueFilterDataSet;
-import org.apache.iotdb.db.query.executor.AggregationExecutor;
-import org.apache.iotdb.db.query.executor.FillQueryExecutor;
-import org.apache.iotdb.db.query.executor.LastQueryExecutor;
-import org.apache.iotdb.db.query.executor.QueryRouter;
-import org.apache.iotdb.db.query.executor.RawDataQueryExecutor;
+import org.apache.iotdb.db.query.executor.*;
```
Comment (wangchao316) [inline cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterQueryRouter.java:35]: generate, we do not use import *.

**Review by wangchao316 — CHANGES_REQUESTED** [review]
Good Job.
you need add some UT or IT, and testcontainers.

File: cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterQueryRouter.java (hunk 1 above)
Comment (liuminghui233) [reply cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterQueryRouter.java:35]: Got it!

**Review by SteveYurongSu — APPROVED** [review]
LGTM. Please cherry-pick it to rel/0.12. 😆

**Pull request merged**